Rectangle/MultiWindow/MultiWindowManager.swift

```diff
@@ -127,18 +142,19 @@ class MultiWindowManager {
         // keep windows with a pid equal to the front window's pid
         var filtered = windows.filter(hasFrontWindowPid(_:))
 
-        var firstSize: CGSize?
+        // parameters for cascading active app windows
+        var caaw: CascadeActiveAppWindows?
 
-        // move the first to become the last
-        if let first = filtered.first, hasFrontWindowPid(first) {
+        if let first = filtered.first {
+            // move the first to become the last (top)
             filtered.removeFirst()
             filtered.append(first)
-            firstSize = first.size
+            caaw = CascadeActiveAppWindows(windowFrame: first.frame, screenFrame: screenFrame, n: filtered.count, size: first.size!, delta: delta)
```

Comment:
If `CascadeActiveAppWindows` becomes `CascadeActiveAppParameters`, then we could change `caaw` to something like `cascadeParameters`